includes/specials/SpecialNewimages.php

```diff
@@ -1,4 +1,8 @@
 <?php
+
+// NOTE: This file is not in use. See WikiaNewFiles.class.php instead
+
+
```

Comment:
We do not have a custom to remove not used MW core files. I'd suggest removing our changes = copying `SpecialNewimages.php` from vanilla MW 1.19.